hal/src/rtl872x/concurrent_hal_impl.h

```diff
@@ -35,8 +35,8 @@ typedef uint8_t os_thread_prio_t;
 /* Default priority is the same as the application thread */
 #define OS_THREAD_PRIORITY_DEFAULT       (2)
 #define OS_THREAD_PRIORITY_CRITICAL      (9)
-#define OS_THREAD_PRIORITY_NETWORK       (7)
-#define OS_THREAD_PRIORITY_NETWORK_HIGH  (8)
+#define OS_THREAD_PRIORITY_NETWORK       (6)
+#define OS_THREAD_PRIORITY_NETWORK_HIGH  (7)
```

Comment:
Do these changes have any effect? Do we know what priority the SDK threads run at? Do we need to take into account their priorities perhaps?

Reply:
This is just fixing the general problem: MAC network threads should be below LwIP TCP/IP thread.